config/Jetpack.alpha.xcconfig

```diff
@@ -2,3 +2,4 @@
 
 BUILD_SCHEME = Jetpack
 WP_PUSH_NOTIFICATION_APP_ID = "com.jetpack.alpha"
+WP_BUILD_CONFIGURATION = alpha
```

Comment:
I'm planning to remove this duplication (WP/JP), but it requires refactoring the .xcconfig file. There has two be only three:

```
Common <- WordPress
Common  <- Jetpack
```